common/djangoapps/enrollment/tests/test_views.py

```diff
@@ -555,6 +556,19 @@ def test_enrollment_throttle_for_user(self):
             expected_status = status.HTTP_429_TOO_MANY_REQUESTS if attempt >= self.rate_limit else status.HTTP_200_OK
             self.assert_enrollment_status(expected_status=expected_status)
 
+    @ddt.data('staff', 'user')
+    def test_enrollment_throttle_is_set_correctly(self, user_scope):
+        """ Make sure throttle rate is set correctly"""
```

Comment:
Nit:
```
""" Make sure throttle rate is set correctly for different user types. """
```
User type?  User scope?  Other?  Not sure what the right name is, but just calling out the ddt which I missed at first.